docs/src/main/sphinx/sql/drop-catalog.md

```diff
@@ -0,0 +1,38 @@
+# DROP CATALOG
+
+## Synopsis
+
+```text
+DROP CATALOG catalog_name
+```
+
+## Description
+
+Drops an existing catalog. Dropping a catalog does not interrupt any running
+queries that use it, but makes it unavailable to any new queries.
+
+:::{warning}
+Some connectors are known not to release all resources when dropping a catalog
+that uses such connector. This includes all connectors that can read data from
```

Comment:
```suggestion
that uses such connector. This includes all connectors, but is not limited to, that can read data from
```

or something to that effect

Reply:
The first sentence in this warning says we haven't tested all connectors, isn't that enough?

Reply:
people read too much into lists. does it hurt being explicit?

Reply:
I don't have a strong opinion on this, except for not using too long sentences :-) I'd defer to @mosabua

Reply:
From what I know the memory leakage is coming from the HDFS/Hadoop libraries only - so only the object storage connectors are affected. Once those are removed as dependencies (very soon) the whole note will be removed.

Reply:
Lets not have this hedging language